CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ### 0.10.1 (Next)
 
-* Your contribution here.
+* [#225](https://github.com/tim-vandecasteele/grape-swagger/pull/225): Fixed param_type to have it read from parameter's documentation hash, e.g. `requires :input, documentation: {param_type: 'query'}` - [@zsxking](https://github.com/zsxking)
```

Comment:
OCD: Missing a period at the end of the sentence. I would remove the whole e.g. thing too, keep it short.